src/vpp_plugins/vpp-chaperone/VppConfigManager.cpp

```diff
@@ -1644,89 +1651,90 @@ void VppConfigManager::doCpePolicerConfig (VppClient &vppClient,
   // otherwise VPP will crash if policed traffic is being passed while the
   // config is deleted.
   bool ifaceStopped = false;
-  // Delete old tables
-  std::vector<u32> tableIds;
-  vppClient.getClassifierTableIds (tableIds);
+  u32 swIfIndex = vppClient.ifaceToVppIndex (interface);
+  u32 tableId;
+  vppClient.getClassifierByIndex (swIfIndex, tableId);
```

Comment:
Check the return value. What does this return if the interface does not have a classifier table? Will the code below crash?